devices/stm32f301.yaml

```diff
@@ -58,6 +63,22 @@ ADC1_2:
       displayName: CCR
       alternateRegister: ""
 
+EXTI:
+  _delete:
+    _interrupts:
+      - UART4_EXTI34
+      - UART5_EXTI35
+  _modify:
+    IMR2:
+      resetValue: "0xFFFFFFFE"
```

Comment:
In RM0136 rev8 14.3.7 it looks like this is a reset value of 0xFFFF_FFFC?

Reply:
RM0316 is for f303. RM0366 rev4 says its 0xFFFF_FFFE.

Reply:
Oh, yep, thanks! I still had 316 open from the last PR, sorry.